tensornetwork/tn_keras/test_layer.py

```diff
@@ -113,7 +114,7 @@ def test_train(dummy_data, make_model):
                 metrics=['accuracy'])
 
   # Train the model for 10 epochs
-  history = model.fit(data, labels, epochs=10, batch_size=32)
+  history = model.fit(data, labels, epochs=50, batch_size=64)
```

Comment:
Do these tests not pass at 10 epochs? 50 seems unnecessary.

Reply:
Only some do. After 50 all tests have consistently decreasing loss

Reply:
The problematic tests were only passing by chance before. If you look at the loss you'll see that it is still fluctuating wildly at 10 epochs.  If just happened so that right at 10 epochs the loss was smaller than the initial value, but e.g. at 11 or maybe 12 it was again larger. The new tf release just exposed that.